paradox/lib/ps.py

```diff
@@ -14,3 +14,6 @@ def sendEvent(event: dict):
 def sendChange(etype: str, label: str, property: str, value: str, initial: bool=False):
     payload = dict(type=etype, label=label, property=property, value=value, initial=initial)
     pub.sendMessage(PREFIX + "changes", change=payload)
+
+def sendInternal(message: str):
```

Comment:
I have just realised that there is a bug. I forgot to add PREFIX. I fixed it in dev.
```
def sendMessage(topicName: str, **msgData):
    pub.sendMessage(PREFIX + topicName, **msgData)
```

Then `sendInternal` is not required.

Reply:
what is PREFIX?  Does this mean the "enumerated_properties" message will not be sent now?

Reply:
So looking at the merged changes it looks like
```
if first_loop:
                first_loop = False
                ps.sendInternal('properties_enumerated')
```
is removed, and I'm using this as a trigger for "I've found all my potential nodes, go process them" and once they're processed I start looking at the changes.

Reply:
This was added to support that functionality and was mandatory for homie (2.0) as we must declare all properties before activating the interface.

Reply:
they are actually all internal. So this is just duplicate function there.

See how I do it for homeassistant:
HASS Interface subscribes to:
```
        ps.subscribe(self._handle_status_update, "status_update")
        ps.subscribe(self._handle_labels_loaded, "labels_loaded")

# Handlers
def _handle_labels_loaded(self, data): # Fires when ALL labels are loaded. data has all labels in groups (partiotions, zones, pgms...)
    logger.debug(data)
def _handle_status_update(self, status): #  Fires when all status responses were received. status has all parsed status responses merged into one dict. Useful for partition arming state detection. See hass interface.
    logger.debug(status)
```
I dispatch status update when all statuses were laoded:
```
# in paradox.py
ps.sendMessage('status_update', status=merged)
# ...
# and for labels in panel.py:
ps.sendMessage('labels_loaded', data=self.core.data)
```